src/controls/widgets/enum/WidgetEnumUI.tsx

```diff
@@ -58,7 +58,15 @@ export const EnumSelectorUI = observer(function EnumSelectorUI_(p: {
     const value = p.value()
     const hasError = Boolean(value.isSubstitute || value.ENUM_HAS_NO_VALUES)
     return (
-        <div tw='flex-1'>
+        <div
+            tw={[
```

Comment:
note: I like those tw arrays: they make it easy to group things that go together on the same line, they make it easy to add comments above lines if we want, etc.
other note: the array elements can also be objects, where every non null values will add the key to the list of classes.